release-notes/docs/3.9.0.md

```diff
@@ -0,0 +1,163 @@
+# 3.9.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [DHIS2 integration](#dhis2-integration)
+- [Supervise target progress](#supervise-target-progress)
+- [RapidPro integration](#rapidpro-integration)
+- [Case management workflows](#case-management-workflows)
+- [Instance monitoring](#instance-monitoring)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+#### Outbound Push only sends each report once
+
+The implementation of Outbound Push has changed as part of [cht-core#6306](https://github.com/medic/cht-core/issues/6306). Now each configured outbound push will be sent only once per doc. Specifically, the first time your `relevant_to` function resolves to `true` is the first and only time a particular record will be send to a configured external service. This is to ensure that pushes are stable and predictable, and additional pushes are not sent unintentionally or unexpectedly. We are working on allowing for multiple pushes, please see [cht-core#6419](https://github.com/medic/cht-core/issues/6419) and contribute your situation to help us determine the best way forward.
+
+#### Cleanup operations may impact app load times
+
+We have implemented a purging function to clean up obsolete feedback and telemetry docs as part of [cht-core#5902](https://github.com/medic/cht-core/issues/5902). This will free up disk space and improve performance over time, however initially may cause slower app load times while the backlog is cleared. The cleanup is capped to 1000 docs per app load to limit the impact on users.
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://docs.communityhealthtoolkit.org/core/overview/supported-software/#dependencies) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## DHIS2 integration
+
+The CHT now supports calculating and exporting data for [DHIS2](https://www.dhis2.org) integration. The Core Framework can now be configured to calculate DHIS2 dataValues, aggregate patient level data, view aggregate data offline, and export the aggregate data formatted for DHIS2. The data is also available via an API so you can access it from an interoperability layer such as OpenHIM.
```

Comment:
I'd love to link to the demo here. Can we publish [this video](https://drive.google.com/file/d/1n_5b3-AdvA0ru57q5Kt6mq5714O6DFoW/view) to YouTube for example?

Reply:
No response...